engine/crates/integration-tests/tests/federation/basic/operation_limits.rs

```diff
@@ -3,7 +3,7 @@ use graphql_mocks::{FakeGithubSchema, MockGraphQlServer};
 use integration_tests::{federation::GatewayV2Ext, runtime};
 
 #[rstest::rstest]
-#[case(
+#[case( // 1
```

Comment:
You know you can give these better names with `case::insert_name_here`?

Reply:
good point, should have done that instead.